### DISCUSSION TITLE
Element offset from top

### DISCUSSION THREAD

**Discussion by pttnx** [idiotWu/smooth-scrollbar#389] [Q&A] [answered]
## Environment
- Browser:
- Version of smooth-scrollbar: 8.x

I need to get the y offset of an element inside the scroll-content.
$(elem).offset().top only gets current position in viewport.

Thanks!

**Comment by dolphin-wood** [comment, marked as answer]
Try:

```js
elem.getBoundingClientRect().top - scrollbar.contentEl.getBoundingClientRect().top
```

no jQuery is needed.

**Reply by pttnx** [reply]
thanks a lot! works as expected
